apps/music/style/music.css

```diff
@@ -1006,14 +1006,8 @@ html[dir="rtl"] li.list-item {
 
 li.list-item a {
   display: block;
-  color: #fff;
   text-decoration: none;
-  outline: none;
   margin: 0;
-  text-shadow: black 0.1rem 0.1rem;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
```

Comment:
I couldn't find anywhere in the application where there is a child `<a>` in a `.list-item`. So I re-appropriated that rule and cleaned it up for the case above.